<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Trapnell lab / projects</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/icon.jpg"/>	
			
	<!-- Google Analytics -->	
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-58618307-1', 'aeteschendorff-lab.github.io');
		ga('send', 'pageview');

	</script>
			
</head>

<body>
	
	<div id="header">
		<nav class="navbar navbar-default navbar-static-top" role="navigation">	 
			<div class="container">	
				
				<div class="navbar-header">
					<object class="pull-left logo" data="/images/icon.jpg" ></object>
					<a class="navbar-brand" href="/">Teschendorff Lab</a>
				</div>
				
				<ul class="nav navbar-nav navbar-right">
                    <!-- 
                    <li>
                    
                    <a href="/blog/">blog</a>
                    </li> -->
					
					<li>
					
					<a href="/papers/">Papers</a></li>
					
					<li class="active">
					
					<a href="/projects/">Projects</a></li>	  
					
					<li>
					
					<a href="/team/">People</a></li>	  	  
				</ul>
			</div>
		</nav>	
	</div>
	
	<div class="container">	
		
	<div class="container" style="max-width: 950px">

  <div class="smallhead">The <strong>mission</strong> of the Teschendorff is to... Our work towards these goals spans three areas illustrated below.</div>
    <div class="bigspacer"></div>
    <center>
    <img src="" alt="..." class="img-responsive" style="width:75%;">
    </center>
    <div class="bigspacer"></div>
  <div class="smallhead">Our <strong>research strategy</strong> is to </div>

    <div class="bigspacer"></div>
    <p class="lead">You can learn more about specific projects below:
    <div class="smallspacer"></div>
</div>

<div class="container">

    <div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>EpiDISH (Epigenetic Dissection of Intra Sample Heterogeneity) </b></h3>
      <br>
            <p> The EpiDISH package provides tools to infer the proportions of a priori known cell subtypes present in a sample representing a mixture of such cell-types. Inference proceeds via one of 3 methods (Robust Partial Correlations-RPC(Teschendorff et al. 2017), Cibersort (CBS)(Newman et al. 2015), Constrained Projection (CP)(Houseman et al. 2012)), as determined by user. Besides, we also provide a method - CellDMC that allows the identification of differentially methylated cell-types and their directionality of change. For now, the package contains 4 references, including two blood subtypes reference, as well as one reference with epithelial cells, fibroblasts, and total immune cells, and one reference with epithelial cells, fibroblasts, fat cells, and total immune cells, described in (Teschendorff et al. 2017) and (Zheng, Webster, et al. 2018).</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/epidish/epidish.jpg" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseepidish" aria-expanded="true" aria-controls="collapseepidish" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseepidish" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <p><strong><a href="https://bioconductor.org/packages/release/bioc/html/EpiDISH.html">[BioConductor Link]</a></strong></p>
<p><strong>Maintainer</strong>: Shijie Charles Zheng</p>
<p><strong>Citation</strong> </p>
<p>Teschendorff, Andrew E, Charles E Breeze, Shijie C Zheng, and Stephan Beck. 2017. “A comparison of reference-based algorithms for correcting cell-type heterogeneity in Epigenome-Wide Association Studies.” BMC Bioinformatics 18 (1):105.</p>
<p>Teschendorff, Andrew E, and Shijie C Zheng. 2017. “Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.” Epigenomics 9 (5):757–68.</p>
<p>Zheng, Shijie C, Charles E Breeze, Stephan Beck, and Andrew E Teschendorff. 2018. “Identification of differentially methylated cell-types in Epigenome-Wide Association Studies.” Nature Methods 15 (12):1059–66.</p>
<p>Zheng, Shijie C, Amy P Webster, Danyue Dong, Andy Feber, David G Graham, Roisin Sullivan, Sarah Jevons, et al. 2018. “A novel cell-type deconvolution algorithm reveals substantial contamination by immune cells in saliva, buccal and cervix.” Epigenomics 10 (7):925–40.</p>
<p><strong>Installation</strong></p>
<p>To install this package, start R (version "3.6") and enter:</p>
<pre><code>if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("EpiDISH", version = "3.9")</code></pre>
<p><strong>Documentation</strong></p>
<p>To view documentation for the version of this package installed in your system, start R and enter:</p>
<pre><code>browseVignettes("EpiDISH")</code></pre>
<p><strong><a href="https://bioconductor.org/packages/devel/bioc/vignettes/EpiDISH/inst/doc/EpiDISH.html">HTML</a></strong>: EpiDISH Vignette</p>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>




    <div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>SEPIRA (Systems EPigenomics Inference of Regulatory Activity) </b></h3>
      <br>
            <p> SEPIRA is a novel algorithm which estimates transcription factor activity in any given sample from its genome-wide mRNA expression or DNA methylation profile(Chen Y, Widschwendter M, and Teschendorff AE 2017). It encompasses two main steps: 1.Construction of a tissue-specific transcription factor regulatory network, consisting of transcription factors that are more highly expressed in the user-specified tissue type (the ‘tissue type of interest’) compared to other tissue types, plus an associated set of high-confidence downstream targets. 2.Estimation of transcription factor activity in this network, in any given dataset consisting of gene expression or promoter DNA methylation profiles.</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/SEPIRA/SEPIRA.png" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseSEPIRA" aria-expanded="true" aria-controls="collapseSEPIRA" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseSEPIRA" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <p><strong><a href="https://bioconductor.org/packages/release/bioc/html/SEPIRA.html">[BioConductor Link]</a></strong></p>
<p><strong>Maintainer</strong>: Yuting Chen</p>
<p><strong>Installation</strong></p>
<p>To install this package, start R (version "3.5") and enter:</p>
<pre><code>if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("SEPIRA", version = "3.8"</code></pre>
<p><strong>Documentation</strong></p>
<p>To view documentation for the version of this package installed in your system, start R and enter:</p>
<pre><code>browseVignettes("SEPIRA")</code></pre>
<p><strong><a href="https://bioconductor.org/packages/release/bioc/vignettes/SEPIRA/inst/doc/SEPIRA.html">HTML</a></strong>: Introduction to SEPIRA</p>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>


<div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>ChAMP (The Chip Analysis Methylation Pipeline)</b></h3>
      <br>
            <p> The ChAMP package is designed for the analysis of Illumina Methylation beadarray data (EPIC and 450k) and provides a pipeline that integrates currently available 450k and EPIC analysis methods. The new version of ChAMP, extends and improves this analysis pipeline, adding novel and enhanced functionalities, including detection of differentially methylated genomic blocks (DMB), Gene Set Enrichment Analysis (GSEA), a variety of methods for correcting cell-type heterogeneity and detection of differentially methylated gene modules. Notably, the new package provides a series of web-based graphical user interfaces (GUIs), which facilitate analyses and enhance user-experience.</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/ChAMP/ChAMP.png" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseChAMP" aria-expanded="true" aria-controls="collapseChAMP" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseChAMP" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <p><strong><a href="https://bioconductor.org/packages/release/bioc/html/ChAMP.html">[BioConductor Link]</a></strong></p>
<p><strong>Maintainer</strong>: Yuan Tian <champ450k at="" gmail.com=""></champ450k></p>
<p><strong>Citation</strong> (from within R, enter citation("ChAMP")):</p>
<p>Yuan Tian, Tiffany J Morris, Amy P Webster, Zhen Yang, Stephan Beck, Andrew Feber, Andrew E Teschendorff; ChAMP: Updated Methylation Analysis Pipeline for Illumina BeadChips, Bioinformatics, , btx513, <a href="https://doi.org/10.1093/bioinformatics/btx513">https://doi.org/10.1093/bioinformatics/btx513</a></p>
<p><strong>Installation</strong></p>
<p>To install this package, start R and enter:</p>
<pre><code>source("http://bioconductor.org/biocLite.R")
biocLite("ChAMP")</code></pre>
<p><strong>Documentation</strong></p>
<p>To view documentation for the version of this package installed in your system, start R and enter:</p>
<pre><code>browseVignettes("ChAMP")</code></pre>
<p><strong><a href="https://www.bioconductor.org/packages/devel/bioc/vignettes/ChAMP/inst/doc/ChAMP.html">HTML</a></strong>: ChAMP Tutorial</p>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>

        <div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>FEM (Functional Epigenetic Modules) algorithm</b></h3>
      <br>
            <p>FEM can dentify interactome hotspots of differential promoter methylation and differential ex-pression, where an inverse association between promoter methylation and gene expression is assumed.A R package to identify interactome hotspots of differential promoter methylation and differential expression, where an inverse association between promoter methylation and gene expression is assumed1.</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/FEM/pic.png" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <p><strong><a href="http://www.bioconductor.org/packages/release/bioc/html/FEM.html">[Bioconductor Link]</a></strong></p>
<p><strong>Installation</strong></p>
<p>To install this package, start R and enter:</p>
<pre><code>source("http://bioconductor.org/biocLite.R")
biocLite("FEM")</code></pre>
<p><strong>Documentation</strong></p>
<p>To view documentation for the version of this package installed in your system, start R and enter:</p>
<pre><code>browseVignettes("FEM")</code></pre>
<p><strong><a href="/images/FEM/intro.pdf">PDF</a></strong> : A R package to identify interactome hotspots of differential promoter methylation and differential expression, where an inverse association between promoter methylation and gene expression is assumed1.</p>
<p><strong><a href="/images/FEM/manual.pdf">PDF</a></strong>: Reference Manual</p>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>

  <div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>BMIQ (Beta MIxture Quantile dilation) algorithm</b></h3>
      <br>
            <p>A novel model-based intra-array normalisation strategy for 450k data, called BMIQ (Beta MIxture Quantile dilation), to adjust the beta-values of type2 design probes into a statistical distribution characteristic of type1 probes.  BMIQ will be useful as a preprocessing step for any study using the Illumina Inﬁnium 450k platform.</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/BMIQ/pic.png" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="true" aria-controls="collapseTwo" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseTwo" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <h3>BMIQ</h3>
<p><strong><a href="/images/BMIQ/BMIQ_1.4.R">R Script for BMIQ</a></strong> This is the script of BMIQ</p>
<p><strong><a href="/images/BMIQ/DoBMIQ.R">R Script for DoBMIQ</a></strong> This is a script to run BMIQ on a set of Methylation Data.</p>
<p>To run BMIQ code, after placing all stuffs(data, probe information, annotation) in one folder, you can use following R code to calculate it out:</p>
<pre><code>source("DoBMIQ.R")</code></pre>
<p><strong>Or you can find BMIQ function in ChAMP Package, which is developed by Tiffany Morris</strong></p>
<p><strong>Installation</strong>
To install ChAMP package, start R and enter:</p>
<pre><code>source("http://bioconductor.org/biocLite.R")
biocLite("ChAMP")</code></pre>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>
            
            
            <div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>ISVA (Independent Surrogate Variable Analysis)  algorithm</b></h3>
      <br>
            <p> We present a modified SVA, called Independent Surrogate Variable Analysis (ISVA), to identify features correlating with a phenotype of interest in the presence of potential confounding factors. ISVA should be useful as a feature selection tool in studies that are subject to confounding.</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/ISVA/pic.jpg" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="true" aria-controls="collapseThree" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseThree" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <p><strong><a href="http://cran.r-project.org/web/packages/isva/index.html">[CRAN Link]</a></strong></p>
<p>isva: Independent Surrogate Variable Analysis</p>
<p>Independent Surrogate Variable Analysis is an algorithm for feature selection in the presence of potential confounding factors.</p>
<p>Version:    1.8
Depends:    qvalue, fastICA
Published:    2013-11-04
Maintainer:    Andrew Teschendorff <a.teschendorff at="" ucl.ac.uk="">
License:    GPL-2
NeedsCompilation:    no
CRAN checks:    isva results
<strong><a href="/images/ISVA/manual.pdf">PDF</a></strong>:  Reference Manual</a.teschendorff></p>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>
            

            
            
            
    <div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>DART (Denoising Algorithm using Relevance network Topology) algorithm</b></h3>
      <br>
            <p> Denoising Algorithm based on Relevance network Topology (DART) is an algorithm designed to evaluate the consistency of prior information molecular signatures (e.g in-vitro perturbation expression signatures) in independent molecular data (e.g gene expression data sets). If consistent, a pruning network strategy is then used to infer the activation status of the molecular signature in individual samples.</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/DART/pic.png" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="true" aria-controls="collapseFour" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseFour" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <p><strong><a href="http://www.bioconductor.org/packages/release/bioc/html/DART.html">[BioConductor Link]</a></strong></p>
<p><strong>Maintainer</strong>: Katherine Lawler <katherine.lawler at="" kcl.ac.uk=""></katherine.lawler></p>
<p><strong>Citation</strong> (from within R, enter citation("DART")):</p>
<p>Y J, K L, GS P, A P and al. e (2011). “DART: Denoising Algorithm based on Relevance network Topology improves molecular pathway activity inference.” BMC Bioinformatics, 12, pp. 403.</p>
<p><strong>Installation</strong></p>
<p>To install this package, start R and enter:</p>
<pre><code>source("http://bioconductor.org/biocLite.R")
biocLite("DART")</code></pre>
<p><strong>Documentation</strong></p>
<p>To view documentation for the version of this package installed in your system, start R and enter:</p>
<pre><code>browseVignettes("DART")</code></pre>
<p><strong><a href="/images/DART/intro.pdf">PDF</a></strong>: DART Tutorial</p>
<p><strong><a href="/images/DART/manual.pdf">PDF</a></strong>: Reference Manua</p>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>
            
            
            <div class="bs-callout bs-callout-danger" style="border:4px solid #333; ">    
       <div class="row">
          <div class="col-lg-7" style="display: table-cell;vertical-align:middle;">
              <h3><b>Signalling Entropy</b></h3>
      <br>
            <p> we will describ e one approach which do es it at the level of individual samples, in which case the gene expression data must be p ositively valued (e.g. the expression data must derive from Affy, Illumina or RNA-Seq count data) and the mass action principle is used to construct the sto chastic matrix over the network..</p>
            <br>
          </div><!-- /.col-lg-4 -->
          <div class="col-lg-5" style="font-weight:bold">
              <h3><b></b></h3>
              <img src="/images/SignalEntropy/pic.png" alt="examiner" height="250px">
          </div><!-- /.col-lg-4 -->
        </div><!-- /.row -->
       <br>
      <div class="panel panel-default">
        <div class="panel-heading" role="tab" id="headingOne">
          <h4 class="panel-title">
            <a data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="true" aria-controls="collapseFive" style="color:#333">
              Download and Install
            </a>
          </h4>
        </div>
        <div id="collapseFive" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
          <div class="panel-body">
              
              
              <!--detailed information-->
              <p><strong><a href="http://sourceforge.net/projects/signalentropy/files/?source=navbar">[SourceForge Link]</a></strong></p>
<p><strong>DESCRIPTION OF FILES:</strong></p>
<p><strong>1) dataSCM2.Rd:</strong> R data object file containing gene expression data matrix, avdataSCM2.m, from the stem cell matrix 2 compendium. Matrix is of size ~17000 genes X 191 samples. The type of cell (i.e hESC=1, iPSC=2 or somatic differentiated=3) is indicated in the phenoSCM2.lv[[1]] entry. Rownames of expression data matrix annotated to Entrez gene IDs.</p>
<p><strong>2) hprdAsigH-13Jun12.Rd:</strong> R data object file with a model of PPI network specified by the adjacency matrix hprdAsigH.m. Rownames annotated to Entrez gene IDs.</p>
<p><strong>3) sr.Rd:</strong> an auxiliary R object file needed to run the vignette.</p>
<p><strong>4) DoIntegPIN.R:</strong> R-function to perform integration of a PPI network with a gene expression data matrix, extracting the maximally connected subnetwork.</p>
<p><strong>5) CompSR.R:</strong> R-function to compute the entropy rate plus auxiliary functions.</p>
<p><strong>6) CompSRana.R:</strong> R-function to compute the entropy rate analytically assuming detgailed balance (this will speed up the computation).</p>
<p><strong>7) <a href="/images/SignalEntropy/intro.pdf">sigent.pdf:</a></strong> vignette/manual pdf.</p>              <!--detailed information over-->   
          </div>
        </div>
      </div>
  </div>
            
</div>
<!---------------------------------------end--------------------------------------->






<!--
<div class="bigspacer"></div>
<center>
<div class="title">
Biology
</div>
</center>
<div class="bigspacer"></div>

	

	

	

	

	

	

	

	

	

	

	

	

</div>
 -->

	
	</div>
		
	<div id="footer"><span style="display:none">foo</span></div>
	    		
	<!-- MathJax support -->
	<script type="text/javascript"
	 src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
	<!-- MathJax config -->
	<script src="/js/mathjax.config.js"></script>	
			
</body>
</html>

