## Welcome to Aeteschendorff Lab

We develop and apply advanced statistical methodology to enable a more meaningful interpretation of large scale multi-dimensional cancer-omic data. Specifically, we are applying tools from network physics, graph theory, Bayesian statistics, signal processing and machine learning to help address some of the outstanding challenges in the cancer genomics and epigenomics fields. We are particularly interested in using the language of networks and statistical mechanics to help tame the complexity of multi-dimensional medical genomic data, and to unravel the systems biology principles underlying complex genetic diseases like cancer.



### Recent papers

* Zheng SC, Breeze CE, Beck S, Teschendorff AE. Identification of differentially methylated cell types in epigenome-wide association studies. Nat Methods. (2018) [Click here for more info](https://www.nature.com/articles/s41592-018-0213-x)

* Teschendorff AE. Avoiding common pitfalls in machine learning omic data science. Nat Material. (2018) [Click here for more info](https://doi.org/10.1038/s41563-018-0241-z)

* Teschendorff AE, Relton CL. Statistical and integrative system-level analysis of DNA methylation data. Nat Rev Genet (2018) [Click here for more info](https://www.nature.com/articles/nrg.2017.86.pdf)

* Teschendorff AE, Enver T. Single-cell entropy for accurate estimation of differentiation potency from a cell's transcriptome Nat Commun (2017) [Click here for more info](https://www.nature.com/articles/ncomms15599)

* Zheng SC, et al. Correcting for cell-type heterogeneity in epigenome-wide association studies: revisiting previous analyses Nat Methods (2017) [Click here for more info](http://www.nature.com/nmeth/journal/v14/n3/full/nmeth.4187.html)

* Yang Z, et al. Correlation of an epigenetic mitotic clock with cancer risk Genome Biol (2016) [Click here for more info](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3)

* Zheng SC, Widschwendter M, Teschendorff AE. Epigenetic drift, epigenetic clocks and cancer risk Epigenomics (2016) [Click here for more info](http://www.ncbi.nlm.nih.gov/pubmed/27104983#)

* Teschendorff AE, et al. DNA methylation outliers in normal breast tissue identify field defects that are enriched in cancer Nat Commun (2016) [Click here for more info](http://www.nature.com/ncomms/2016/160129/ncomms10478/full/ncomms10478.html)

* Teschendorff AE, et al. Correlation of Smoking-Associated DNA Methylation Changes in Buccal Cells With DNA Methylation Changes in Epithelial Cancer JAMA Oncol (2015) [Click here for more info](http://oncology.jamanetwork.com/article.aspx?articleid=2293216)

* Yuan T, et al. An integrative multi-scale analysis of the dynamic DNA methylation landscape in aging PLoS Genet (2015) [Click here for more info](http://www.plosgenetics.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pgen.1004996&representation=PDF)

* Jones A, et al. Role of DNA methylation and epigenetic silencing of HAND2 in endometrial cancer development PLoS Med (2013) [Click here for more info](http://www.plosmedicine.org/article/info%3Adoi%2F10.1371%2Fjournal.pmed.1001551)

* West J, Widschwendter M, Teschendorff AE. Distinctive topology of age-associated epigenetic drift in the human interactome Proc Natl Acad Sci U S A. (2013) [Click here for more info](http://www.pnas.org/content/110/35/14138.abstract)

* Teschendorff AE, et al. A beta-mixture quantile normalization method for correcting probe design bias in Illumina Infinium 450 k DNA methylation data Bioinformatics (2013) [Click here for more info](http://bioinformatics.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=23175756)

* Teschendorff AE, et al. Epigenetic variability in cells of normal cytology is associated with the risk of future morphological transformation Genome Med (2012) [Click here for more info](http://genomemedicine.com/content/4/3/24)

* Teschendorff AE, Zhuang J, Widschwendter M. Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies Bioinformatics (2011) [Click here for more info](https://academic.oup.com/bioinformatics/article/27/11/1496/217145)

* Teschendorff AE, et al. Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer Genome Res (2010) [Click here for more info](http://genome.cshlp.org/content/early/2010/03/09/gr.103606.109)



### Research Focus
###### Computational Systems Biology Analysis of Single-Cell data
![img](http://www.picb.ac.cn/compsysg/Main/Fig1scentBR.png)
We are developing novel computational and statistical methods to help analyse and better interpret single-cell omic data, with a particular focus on systems biological aspects.
###### Epigenomics of Ageing, Stem Cells and Cancer
![img](http://www.picb.ac.cn/compsysg/Main/Fig1revFMnew.png)
We are exploring how informative DNA methylation changes that accrue in normal cells as a function of age and exposure to risk factors are for predicting cancer risk. 
###### Cell-type heterogeneity and deconvolution for EWAS
![img](http://www.picb.ac.cn/compsysg/Main/Fig4nm-A.png)
We are exploring and developing statistical cell-type deconvolution methods to dissect cell-type heterogeneity aimed at epigenome studies performed in general complex tissues. 
###### Cancer System-Omics and Network Physics
![img](http://www.picb.ac.cn/compsysg/Main/FigLungNetA.png)
We use innovative strategies drawing on methods from graph theory/network science/signal processing to analyse complex and high-dimensional multi-omic data, with a view to elucidate the systems-biological principles underlying diseases like cancer.
### Support or Contact
For more details see [_our website_](http://www.picb.ac.cn/compsysg/index.php).
![img](http://www.picb.ac.cn/compsysg/Main/pic_3.png)

   <div class="navbar-collapse collapse"> 
     <ul class="nav navbar-nav"> 
      <li class="active"><a href="./index.md">Home</a></li> 
      <li><a href="./Projects.md">Research</a></li> 
      <li><a href="./People.md">People</a></li>
      <li><a href="./Papers">Publications</a></li>
     </ul> 
    </div> 
